helpdesk_mgmt_sla_rule/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class HelpDeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    sla_id = fields.Many2one("helpdesk.sla", string="SLA", compute="_compute_sla")
```

Comment:
missing store=True to be able to search on this field (like you do here https://github.com/OCA/helpdesk/pull/362/files?diff=split&w=0#diff-5e554a183028bdbcbc1eb0a377cf2553784b63af8c77381f454c5ab1eb875c8eR11)